cla_public/static-src/javascripts/modules/address-finder.js

```diff
@@ -73,7 +73,7 @@
       this.addresses = data;
 
       if (data.length < 1) {
-        this.showError('No addresses were found with that postcode');
+        this.showHelp('No addresses were found with that postcode, but you can still enter your address manually');
```

Comment:
There are a number of these in JS, and it doesn't look like they are being translated at the moment.

But yes, they probably should. :-)